.php_cs

```diff
@@ -0,0 +1,26 @@
+<?php
+
+$finder = Symfony\CS\Finder\DefaultFinder::create()
```

Comment:
Will do.

I just remembered this should probably be noted in CONTRIBUTING. Add it to the intro or mention it in a new section after unit testing?